### ISSUE TITLE
Thread summary: Summarise a thread in Sphinx

### ISSUE THREAD

**Issue by tomsmith8** [stakwork/sphinx-mac#200]
thread summary: Pass the text of a thread and summarise it’s current status

- Press summarise button
- Sends content to Stakwork
- Summarises text

Example response: Ed started the thread by asking about a problem with using a twitter API key for Kevin. Ali and Gonzalo tested and the current thinking is that the key is invalid, they are on a call to discuss.

**Comment by uncleJim21** [comment]
I have a prototype I built of this just using ChatGPT. But for privacy I think it's best if we can run through our own private LLM or at least anonymize through a larger anonymity set. Let me know who will be assigned to creating the API and I'll work with them to define requirements @tomsmith8

**Comment by tomsmith8** [comment]
@uncleJim21 yeah agreed. Using our LLaMa model (self-hosted) might make the most sense for privacy reasons. The only downside right now is it takes a little while to process (sometimes a few mins).

Cool will do!
